I've written the tests and I'm _hoping_ they'll work, since I'm running into these errors while trying to make the check target:
```
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libgtest.a(gtest-all.cc.o): undefined reference to symbol 'pthread_getspecific@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
```
and I couldn't figure them out.

--- a/include/stumpless/facility.h
+++ b/include/stumpless/facility.h
@@ -362,11 +362,11 @@ enum stumpless_facility {
  * This function is safe to call from threads that may be asynchronously
  * cancelled.
  *
+ * @since release v2.1.0.
+ *
  * @param facility The facility to get the string from.
  *
  * @return The string representation of the given facility.
- *
- * @since release v2.1.0.
  */
 STUMPLESS_PUBLIC_FUNCTION
 const char *
@@ -385,13 +385,12 @@ stumpless_get_facility_string( enum stumpless_facility facility );
  * This function is safe to call from threads that may be asynchronously
  * cancelled.
  *
+ * @since release v2.1.0.
+ *
  * @param facility_string The facility name to get the enum from.
  *
  * @return The enum integer corresponding to the given facility or -1 if
  * the string is not a valid facility name.
- *
- * TODO: is this correct?
- * @since release v2.1.0.
  */
 STUMPLESS_PUBLIC_FUNCTION
 enum stumpless_facility

--- a/include/stumpless/severity.h
+++ b/include/stumpless/severity.h
@@ -230,13 +230,12 @@ stumpless_get_severity_string( enum stumpless_severity severity );
  * This function is safe to call from threads that may be asynchronously
  * cancelled.
  *
+ * @since release v2.1.0.
+ *
  * @param severity_string The severity name to get the enum from.
  *
  * @return The enum integer corresponding to the given severity or -1 if
  * the string is not a valid severity name.
- *
- * TODO: is this correct?
- * @since release v2.1.0.
  */
 STUMPLESS_PUBLIC_FUNCTION
 enum stumpless_severity

--- a/src/facility.c
+++ b/src/facility.c
@@ -30,7 +30,7 @@ const char *
 stumpless_get_facility_string( enum stumpless_facility facility ) {
   size_t facility_upper_bound = sizeof facility_enum_to_string;
   if ( !facility_is_invalid(facility) ) {
-    return facility_enum_to_string[facility/ sizeof facility_enum_to_string[0]];
+    return facility_enum_to_string[facility >> 3];
   }
   return "NO_SUCH_FACILITY";
 }
@@ -39,7 +39,7 @@ enum stumpless_facility
 stumpless_get_facility_enum( const char *facility_string ) {
   for (int i = 0; i < sizeof facility_enum_to_string / sizeof facility_enum_to_string[0]; i++)
     if (strcmp(facility_string, facility_enum_to_string[i]) == 0)
-      return i * sizeof facility_enum_to_string[0];
+      return i << 3;
   return -1;
 }
 

--- a/src/windows/stumpless.def
+++ b/src/windows/stumpless.def
@@ -187,7 +187,5 @@ EXPORTS
   stumplog_str                                  @172
   stumplog_trace_str                            @173
   stumpless_get_facility_string                 @174
-
-; added in v2.1.0
   stumpless_get_facility_enum                   @175
   stumpless_get_severity_enum                   @176

